Lil-Peep/src/styles/modules/featured-kits.scss

```diff
@@ -84,7 +87,7 @@
 .mid-banner-content {
   display: flex;
   flex-direction: column;
-  height: 300px;
+  height: 280px;
```

Comment:
Properties should be ordered display, flex-direction, font-family, height, justify-content, text-align